template/lambda-sqs-worker-cdk/infra/__snapshots__/appStack.test.ts.snap

```diff
@@ -941,14 +953,26 @@ exports[`returns expected CloudFormation stack for prod 1`] = `
           },
         },
         "FunctionName": "serviceName",
-        "Handler": "node_modules/datadog-lambda-js/dist/handler.handler",
+        "Handler": "/opt/nodejs/node_modules/datadog-lambda-js/handler.handler",
         "KmsKeyArn": {
           "Fn::GetAtt": [
             "kmskey49FBC3B3",
             "Arn",
           ],
         },
         "Layers": [
+          {
+            "Fn::Join": [
+              "",
+              [
+                "arn:aws:lambda:",
+                {
+                  "Ref": "AWS::Region",
+                },
+                ":464622532012:layer:Datadog-Node22-x:126",
```

Comment:
This might become annoying with Renovates to update

Reply:
Yep, I think other parts of the snap would need to be updated when the runtime gets updated anyway